runtime-light/stdlib/curl/curl-context.h

```diff
@@ -0,0 +1,13 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "runtime-core/runtime-core.h"
```

Comment:
Unnecessary include. It's better to just include `cstdint`

Reply:
I don't agree. In any case in the future runtime core will be necessary here